node/grpc/utils.go

```diff
@@ -67,6 +67,12 @@ func GetBlobMessages(in *pb.StoreChunksRequest) ([]*core.BlobMessage, error) {
 
 // GetBlobHeaderFromProto constructs a core.BlobHeader from a proto of pb.BlobHeader.
 func GetBlobHeaderFromProto(h *pb.BlobHeader) (*core.BlobHeader, error) {
+
+	if h == nil {
+		return nil, api.NewInvalidArgError("GetBlobHeaderFromProto: blob header is nil")
```

Comment:
Is this only used to handle API requests at the top level? 
Otherwise, i don't think it should return the api error